testapps/testapp/src/main/java/com/microsoft/identity/client/testapp/Constants.java

```diff
@@ -47,9 +49,15 @@ public static int getResourceIdFromConfigFile(ConfigFile configFile){
             case BROWSER:
                 return R.raw.msal_config_browser;
 
+            case BROWSER_SKIP_BROKER:
+                return R.raw.msal_config_browser_skip_broker;
+
             case WEBVIEW:
                 return R.raw.msal_config_webview;
 
+            case WEBVIEW_SKIP_BROKER:
+                return R.raw.msal_config_webview_skip_broker;
+
```

Comment:
This will result in numerous jsons to be created based on various combinations.
For a more flexible approach we should split the options in its own category e.g. whether or not use Broker . 
This can then be combined with other configuration values and generate the json on fly based on selection.

Reply:
I am fine for now. But for future we should consider this change.